### PR TITLE
Refactor logging config

### PR DESCRIPTION
**Pull request by yakloinsteak**

### CODE REVIEW

**Comment by yakloinsteak** [comment]
@eanders Next week, and after you've peeked at this, I'll test on QA.

File: config/setup_logging.rb [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    config.log_level = ENV.fetch('LOG_LEVEL') { 'info' }.to_sym
+    config.logger = _tagged(ActiveSupport::Logger.new($stdout))
+    config.logger.formatter = OpenPathLogFormatter.new
+    config.lograge.enabled = true
+  end
+
+  def _production
+    # They're the same
+    _staging
+  end
```
Comment (eanders) [inline config/setup_logging.rb]: really minor, but can we put the config in production and reference it from staging?  I like thinking that we care more about what's in production.

**Review by eanders — COMMENTED** [review]
@yakloinsteak , this looks  good.  The only thing I can't seem to tell is if we're preserving the non-JSON format for dev and test.  Give it a run on QA and let's see what we get!

**Comment by yakloinsteak** [comment]
@eanders I'm not in grafana yet, but the cloudwatch logs look fine to me. If the log output is good locally for you on this branch, I think we can merge this after the tests pass.

**Comment by eanders** [comment]
merging.  Thanks!

**Pull request merged**